package.json

```diff
@@ -14,7 +14,7 @@
     "@docusaurus/preset-classic": "^2.0.0-alpha.53",
     "@svgr/webpack": "^5.4.0",
     "classnames": "^2.2.6",
-    "libra-docusaurus": "^0.4.7",
+    "libra-docusaurus": "^0.4.10",
```

Comment:
Let's rename this to libra-components. The name could be confused with a custom implementation of docusaurus.

Reply:
Adding this as a todo, since I feel like it should be it's own commit

Reply:
https://dfuzrdev.atlassian.net/browse/LIB-721